app/views/comments/edit.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```

Comment:
Great job here! You can assign your jquery to variables here to make your code clearer to read!